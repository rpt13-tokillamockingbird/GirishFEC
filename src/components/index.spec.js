import React from 'react';
import { shallow } from 'enzyme';

import Index from './index.jsx';
import { isTSAnyKeyword, exportAllDeclaration } from '@babel/types';

describe('Review index', () => {
  it('Should render App correctly', () => {
    const comp = shallow(<Index />);
    expect(comp).toMatchSnapshot();
  });
});