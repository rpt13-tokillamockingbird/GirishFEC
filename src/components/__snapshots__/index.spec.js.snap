// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review index Should render App correctly 1`] = `ShallowWrapper {}`;
